astropy/table/groups.py

```diff
@@ -34,6 +34,8 @@ def _table_group_by(table, keys):
     grouped_table : Table object with groups attr set accordingly
     """
     from .table import Table
+    from .serialize import _represent_mixins_as_columns
```

Comment:
We may consider exposing the function (but not in this PR).

Reply:
Agreed.  Pylint has started complaining to me about using a private method from another module...